libcontainer/cgroups/fs/pids.go

```diff
@@ -52,11 +53,20 @@ func (s *PidsGroup) GetStats(path string, stats *cgroups.Stats) error {
 		return fmt.Errorf("failed to parse pids.current - %s", err)
 	}
 
-	max, err := getCgroupParamUint(path, "pids.max")
+	maxString, err := getCgroupParamString(path, "pids.max")
 	if err != nil {
 		return fmt.Errorf("failed to parse pids.max - %s", err)
 	}
 
+	var max uint64
```

Comment:
If we want, we can make a value of `-1` mean no limit, but it would mean making that type `int64` (which isn't actually correct).